pkg/kubectl/cmd/convert.go

```diff
@@ -189,12 +189,24 @@ func (o *ConvertOptions) RunConvert() error {
 		if err != nil {
 			return err
 		}
```

Comment:
It's interesting that the code [here](https://github.com/kubernetes/kubernetes/pull/51314/files#diff-c1b6a3c269cbdfd64b769968834ea42eL125) doesn't output anything when invalid groupversion is passed.